### PR TITLE
oadp-1.0: Fix CVE-2023-39325

### PR DESCRIPTION
**Pull request by weshayutin**

### CODE REVIEW

**Review by kaovilai — CHANGES_REQUESTED** [review]
Please add `-buildvcs=false` to Dockerfile
[like so](https://github.com/openshift/openshift-velero-plugin/pull/201/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R22)

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ ENV BUILDTAGS containers_image_ostree_stub exclude_graphdriver_devicemapper excl
 ENV BIN velero-plugins
 COPY --chown=1001 . $APP_ROOT/src/github.com/konveyor/openshift-velero-plugin
 WORKDIR $APP_ROOT/src/github.com/konveyor/openshift-velero-plugin
-RUN go build -installsuffix "static" -tags "$BUILDTAGS" -o _output/$BIN ./$BIN
+RUN go build -buildcvs=false -installsuffix "static" -tags "$BUILDTAGS" -o _output/$BIN ./$BIN
```
Comment (mateusoliveira43) [inline Dockerfile]: typo buildcvs -> buildvcs

Comment (kaovilai) [reply Dockerfile]: take your cvs pill @weshayutin

Comment (weshayutin) [reply Dockerfile]: PXL_20231016_203251374

**Review by kaovilai — APPROVED** [review]
/lgtm

**Pull request merged**